src/App.js

```diff
@@ -1,15 +1,76 @@
+import ForConsumers from './pages/ForConsumers';
+import ForContractors from './pages/ForContractors';
+import LandingPage from './pages/LandingPage';
+import LoggedInConsumer from './pages/LoggedInConsumer';
+import LoggedInContractor from './pages/LoggedInContractor';
 import React from 'react';
+import { useAuth0 } from '@auth0/auth0-react';
 import { Switch, BrowserRouter as Router, Route } from 'react-router-dom';
-import LandingPage from './pages/LandingPage';
 
```

Comment:
im not sure if there is a preferred organization to these that is typical. i set up eslint so it alphabetically organizes, as well as organizes by number of imports on the given line